module/zcommon/zpool_prop.c

```diff
@@ -470,6 +477,10 @@ vdev_prop_init(void)
 	zprop_register_index(VDEV_PROP_TRIM_SUPPORT, "trim_support", 0,
 	    PROP_READONLY, ZFS_TYPE_VDEV, "on | off", "TRIMSUP",
 	    boolean_table, sfeatures);
+	zprop_register_index(VDEV_PROP_SCHEDULER, "scheduler",
+	    VDEV_SCHEDULER_AUTO, PROP_DEFAULT, ZFS_TYPE_VDEV,
+	    "auto | classic | none", "IO_SCHEDULER", vdevschedulertype_table,
+	    sfeatures);
```

Comment:
This should be moved in to the "default index properties" section below.  It looks like failfast somehow managed to end up in the wrong section.   We should fix that up too, but that can be done in it's own PR.